### ISSUE TITLE
MemE v4.0.0 breaks Hugo shortcodes

### ISSUE THREAD

**Issue by reuixiy** [reuixiy/hugo-theme-meme#50]

**Comment by reuixiy** [comment]
Sadly, this bug cannot be fixed temporarily. I will fix this once https://github.com/gohugoio/hugo/issues/6544 is resolved.

To implement the compatibility with LaTeX, I use [`.RawContent`](https://github.com/reuixiy/hugo-theme-meme/blob/e6105a99bab433dffb3a9593fbfbce68ed1539f0/layouts/partials/components/content.html#L2) then [`markdownify`](https://github.com/reuixiy/hugo-theme-meme/blob/c7ce9bbbb0f3f84ae638ee3e041cddd016a867f8/layouts/partials/components/markdownify.html#L43) it, instead of using rendered `.Content`. However, there is [no such function](https://github.com/gohugoio/hugo/issues/3821) for rendering shortcode, which means shortcode won't be rendered.

If you need Hugo shortcodes more than the compatibility with LaTeX, here is the workaround:

1. Copy [content.html](https://github.com/reuixiy/hugo-theme-meme/blob/master/layouts/partials/components/content.html) to `~/blog/layouts/partials/components/content.html` and modify `.RawContent` to `.Content`.

2. Copy [markdownify.html](https://github.com/reuixiy/hugo-theme-meme/blob/master/layouts/partials/components/markdownify.html) to `~/blog/layouts/partials/components/markdownify.html` and modify [these codes](https://github.com/reuixiy/hugo-theme-meme/blob/c7ce9bbbb0f3f84ae638ee3e041cddd016a867f8/layouts/partials/components/markdownify.html#L41-L44) to the following:

    ```html
    <!-- Markdown -->
    {{- $Content := $Deliver.Scratch.Get "Content" -}}
    {{- if not $isContent -}}
        {{- $Content := $Content | markdownify -}}
        {{- $Deliver.Scratch.Set "Content" $Content -}}
    {{- end -}}
    ```

**Comment by MatNoble** [comment]
> Sadly, this bug cannot be fixed temporarily. I will fix this once [gohugoio/hugo#6544](https://github.com/gohugoio/hugo/issues/6544) is resolved.
> 
> To implement the compatibility with LaTeX, I use [`.RawContent`](https://github.com/reuixiy/hugo-theme-meme/blob/e6105a99bab433dffb3a9593fbfbce68ed1539f0/layouts/partials/components/content.html#L2) then [`markdownify`](https://github.com/reuixiy/hugo-theme-meme/blob/c7ce9bbbb0f3f84ae638ee3e041cddd016a867f8/layouts/partials/components/markdownify.html#L43) it, instead of using rendered `.Content`. However, there is [no such function](https://github.com/gohugoio/hugo/issues/3821) for rendering shortcode, which means shortcode won't be rendered.
> 
> If you need Hugo shortcodes more than the compatibility with LaTeX, here is the workaround:
> 
> 1. Copy [content.html](https://github.com/reuixiy/hugo-theme-meme/blob/master/layouts/partials/components/content.html) to `~/blog/layouts/partials/components/content.html` and modify `.RawContent` to `.Content`.
> 2. Copy [markdownify.html](https://github.com/reuixiy/hugo-theme-meme/blob/master/layouts/partials/components/markdownify.html) to `~/blog/layouts/partials/components/markdownify.html` and modify [these codes](https://github.com/reuixiy/hugo-theme-meme/blob/c7ce9bbbb0f3f84ae638ee3e041cddd016a867f8/layouts/partials/components/markdownify.html#L41-L44) to the following:
>    ```
>    
>    {{- $Content := $Deliver.Scratch.Get "Content" -}}
>    {{- if not $isContent -}}
>        {{- $Content := $Content | markdownify -}}
>        {{- $Deliver.Scratch.Set "Content" $Content -}}
>    {{- end -}}
>    ```

According to your prompt, I successfully implemented shortcodes
[Shortcodes based on MemE](https://matnoble.me/posts/shortcodes-practice-tutorial-for-hugo/)

**Comment by reuixiy** [comment]
Yep, Hugo is quite powerful and flexible! Thanks for your sharing :+1:
